src/pages/searching.tsx

```diff
@@ -96,15 +96,23 @@ const Searching = () => {
     return () => {
       socket?.off(events.NEW_EVENT_STOP_SEARCH_SUCCESS);
     };
-  }, [app.currentChatId, authState.email, authState.loginId, router, socket]);
+  }, [app.currentroom, authState.email, authState.loginId, router, socket]);
 
   useEffect(() => {
-    if (!app.currentChatId) {
+    if (!app.currentroom) {
       return;
     }
 
-    router.push('/anonymous')
-  }, [app.currentChatId])
+    router.push('/anonymous');
+  }, [app.currentroom]);
+
+  useEffect(() => {
+    if (!app.currentroom) {
+      return;
+    }
+
+    router.push('/anonymous');
+  }, [app.currentroom]);
```

Comment:
This is a repitition of line (101 - 107)

Reply:
thanks didn't notice while merging